bricks/basic/src/dynamic-form-item/index.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import { EventEmitter, createDecorators } from "@next-core/element";
+import { wrapBrick } from "@next-core/react-element";
+import type { Button, ButtonProps } from "../button/index.js";
+import type {
+  GeneralIcon,
+  GeneralIconProps,
+} from "@next-bricks/icons/general-icon";
+import classNames from "classnames";
+import { FormItemElement } from "../form-item/FormItemElement.js";
+import type {
+  Form,
+  FormEvents,
+  FormMapEvents,
+  FormProps,
+} from "../form/index.js";
+import type { FormItem, FormItemProps } from "../form-item/index.jsx";
+import { ReactUseMultipleBricks } from "@next-core/react-runtime";
+import { UseSingleBrickConf } from "@next-core/types";
+import styleText from "./dynamic-form-item.shadow.css";
+import "@next-core/theme";
+
+const { defineElement, property, event } = createDecorators();
+
+const WrappedIcon = wrapBrick<GeneralIcon, GeneralIconProps>(
+  "icons.general-icon"
+);
+
+const WrappedButton = wrapBrick<Button, ButtonProps>("basic.general-button");
+
+const WrappedForm = wrapBrick<Form, FormProps, FormEvents, FormMapEvents>(
+  "basic.general-form",
+  {
+    onValuesChange: "values.change",
+    onValidateSuccess: "validate.success",
+    onValidateError: "validate.error",
+  }
+);
+
+const WrappedFormItem = wrapBrick<FormItem, FormItemProps>(
+  "basic.general-form-item"
+);
+
+type DynamicFormValuesItem = Record<string, any>;
+
+interface DynamicFormItemProps extends FormItemProps {
+  form?: Form;
+  useBrick?: UseSingleBrickConf[];
+  value?: DynamicFormValuesItem[];
+  validateState?: string;
+  needValidate?: boolean;
+  onChange?: (value: DynamicFormValuesItem[]) => void;
+  onValuesChange?: (values: DynamicFormValuesItem[]) => void;
+}
+
+/**
+ * @id basic.dynamic-form-item
+ * @name basic.dynamic-form-item
+ * @docKind brick
+ * @description 动态表单
+ * @author sailor
+ * @noInheritDoc
+ */
+@defineElement("basic.dynamic-form-item", {
+  styleTexts: [styleText],
+})
+class DynamicFormItem extends FormItemElement {
+  @property() accessor name: string | undefined;
+  @property() accessor label: string | undefined;
+  /**
+   * @default
+   * @required
+   * @description
+   */
+  @property({
+    type: Boolean,
+  })
+  accessor required: boolean | undefined;
+
+  @property({
+    attribute: false,
+  })
+  accessor value: DynamicFormValuesItem[] | undefined;
+
+  /**
+   * @default
+   * @required
+   * @description
+   */
+  @property({
+    attribute: false,
+  })
+  accessor useBrick: UseSingleBrickConf[] | undefined;
```

Comment:
使用 `UseBrickConf` 支持单个对象或数组形式的 useBrick

```suggestion
  accessor useBrick: UseBrickConf | undefined;
```